minitrace-macro/Cargo.toml

```diff
@@ -20,3 +20,10 @@ syn = { version = "1.0.84", features = ["full", "parsing", "extra-traits", "proc
 quote = "1"
 proc-macro2 = "1"
 proc-macro-error = "1"
+
+[dev-dependencies]
+trybuild = "1"
+minitrace = "0.4.0"
+
+[patch.crates-io]
+minitrace = { path = "../" }
```

Comment:
```suggestion
minitrace = { path = "../" }

```

Reply:
Thanks for taking the time to review these changes.

I am not sure I follow this suggested change. 

To clarify, you prefer not to use Cargo's `[patch.cargo.io]` interface, and would rather replace:

```toml
[dev-dependency]
...
minitrace = 0.4.0
...
```

with 

```toml
[dev-dependency]
...
minitrace = { version = "0.4.0", path = "../"}
...
```

Correct?

Reply:
OK, I see the patch is being ignored:

https://github.com/tikv/minitrace-rust/runs/5625725597?check_suite_focus=true#step:4:11